pytorch/buildspec-neuron.yml

```diff
@@ -2,7 +2,7 @@
   region: &REGION <set-$REGION-in-environment>
   framework: &FRAMEWORK pytorch
   version: &VERSION 1.5.1
-  os_version: &OS_VERSION ubuntu16.04
+  os_version: &OS_VERSION ubuntu18.04
```

Comment:
In order to be able to build both ubuntu16.04 and ubuntu18.04 dockerfiles for PT 1.5.1, please move this line from Line 5 to between Line 33 and Line 34 as `ubuntu16.04`, and create another image-entry, similar to `BuildNeuronPTInferencePy3DockerImage` (which you could name `BuildNeuronPTInferencePy3UL18DockerImage`), which has `os_version: &OS_VERSION ubuntu18.04`.